webapp/crashstats/settings/base.py

```diff
@@ -490,6 +490,9 @@ def filter(self, record):
     doc="Make this unique, and don't share it with anybody. It cannot be blank.",
 )
 
+CSRF_COOKIE_NAME = "crashstatscsrfcookie"
+CSRF_USE_SESSIONS = True
```

Comment:
The first item helps us validate things. The second item fixes the bug.

Reply:
I know this PR already merged, but IIUC, the only purpose of `CSRF_COOKIE_NAME` is to validate this change as part of the PR review. Why not set it as a temporary env var in one's local config instead of permanently adding a new env var to settings?

Reply:
The `CSRF_COOKIE_NAME` is a Django setting used by the build-in CSRF middleware and it defaults to `"csrftoken"`. All I did here was change the value to `"crashstatscsrfcookie"`. This doesn't introduce a new environment variable or otherwise affect things.

We can take the `CSRF_COOKIE_NAME` setting out now that this has landed, been pushed to production, and verified.